File: Desktop/PhaseGate/Day 6/Python/Age.py
import datetime
def ages():
	dob = ""
	while len(dob)!= 8:
		person = input(" Enter date of birth in MM-DD-YYYY")
		dob = person.replace("-", "")
				
		dob = person[:2]
		day = int(dob)
		month = person[2:4]
		months = int(month)
		years = person[4:8]
		year = int(years)
		today = datetime.date()
		currentYear = today.year
		currentMonth = today.month
		currentDay = today.day
		age = " "
		if currentDay < day or currentMonth < months:
			age = currentYear - year - 1
		else:
			age = currentYear - year
		print(age)

ages()
		 

File: Desktop/PhaseGate/Day 6/java/Age.java
ngimport java.util.*;
import java.time.*;

public class Age {
public static void main(String[] args) {

Scanner scanner = new Scanner(System.in);
	String userInput = " ";
	do{
	System.out.println("Enter Date Of Birth in DD-MM-YYYY");
	String dob = scanner.nextLine();
	userInput = dob.replace("-", "");
	}while(userInput.length() != 8);
	String result = date(userInput);
	System.out.print(result);
	}



	public static String date(String userInput) {
	String strDay = userInput.substring(0,2);
	int intDay = Integer.parseInt(strDay);
	String strMonth = userInput.substring(2,4);
	int intMonth = Integer.parseInt(strMonth);
	String strYear = userInput.substring(4,8);
	int intYear = Integer.parseInt(strYear);
	LocalDate today = LocalDate.now();
	int todayYear = today.getYear();
	Month month = today.getMonth();
	int todayMonth = month.getValue();
	LocalDateTime now = LocalDateTime.now();
	
	

	int age = 0;
	if (todayMonth < intMonth){
		age = todayYear - intYear - 1;
	}else{
		age = todayYear - intYear;
	}
	String strAge = Integer.toString(age);
	
return strAge;
}
}






 

